css/style.css

```diff
@@ -0,0 +1,94 @@
+* {
+    margin: 0;
+    border: 0;
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #eee;
+}
+
+#header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 1rem 2rem;
+    position: fixed;
+    width: auto;
+    background-color: #eee;
```

Comment:
This will print out [linters](https://github.com/microverseinc/linters-config) errors. Be sure to add linters configuration and pass their checks. They'll help getting your code cleaner automatically :broom: :dash:

Reply:
> This will print out [linters](https://github.com/microverseinc/linters-config) errors. Be sure to add linters configuration and pass their checks. They'll help getting your code cleaner automatically 🧹 💨

- [ ] Make sure that you left the comment under a line or block of code that fits the context the comment is about.

Reply:
> This will print out [linters](https://github.com/microverseinc/linters-config) errors. Be sure to add linters configuration and pass their checks. They'll help getting your code cleaner automatically 🧹 💨

- [ ] I would kindly suggest you to phrased the comments politely, as suggestions instead of orders. For example:

* This is a polite suggestion: *"I think it would be better to use semantic tags"*